### DISCUSSION TITLE
RPC failed; HTTP 400 error

### DISCUSSION THREAD

**Discussion by legyul** [alshedivat/al-folio#2850] [Q&A] [answered]
Thank you for making this awesome template and I am using it well.

While pushing, I got an error as the following.

error: RPC failed; HTTP 400 curl 22 The requested URL returned error: 400
send-pack: unexpected disconnect while reading sideband packet
Writing objects: 100% (15/15), 1.94 MiB | 8.20 MiB/s, done.
Total 15 (delta 6), reused 0 (delta 0), pack-reused 0
fatal: the remote end hung up unexpectedly
Everything up-to-date

How could I fix this error?

**Comment by george-gca** [comment, marked as answer]
This is not a template issue, but a git issue. See if [stackoverflow](https://stackoverflow.com/questions/62753648/rpc-failed-http-400-curl-22-the-requested-url-returned-error-400-bad-request) helps.

**Reply by legyul** [reply]
Thank you so much!
